hudi-client/src/main/java/org/apache/hudi/config/HoodieIndexConfig.java

```diff
@@ -77,6 +77,17 @@
   public static final String BLOOM_INDEX_INPUT_STORAGE_LEVEL = "hoodie.bloom.index.input.storage.level";
   public static final String DEFAULT_BLOOM_INDEX_INPUT_STORAGE_LEVEL = "MEMORY_AND_DISK_SER";
 
+  /**
+   * Only applies if index type is GLOBAL_BLOOM.
+   * <p>
+   * When set to true, an update including the partition path of a record that already exists will result in inserting
```

Comment:
minor: "when set to true, an update to a record with diff partition compared to its existence, will insert the record to new partition and delete from old partition. When set to false, record will be updated to old partition".

Reply:
> can we also add this to the configurations md file , to keep the docs upto date?

The docs update PR is in #1190 . I'll update it once this is merged.